safedelete/managers.py

```diff
@@ -100,6 +101,32 @@ def all(self, **kwargs):
             qs._safedelete_force_visibility = force_visibility
         return qs
 
+    def update_or_create(self, defaults=None, **kwargs):
+        """Regular update_or_create() fails on soft-deleted, existing record with unique constraint on non-id field
+
+        Args:
```

Comment:
I'd refer to the `Manager.update_or_create` method instead.

Reply:
You could say with Sphinx something like this iirc:

```python
def update_or_create(self, defaults=None, **kwargs):
"""See :func:`~django.db.models.Query.update_or_create.`.

Explain what this functionality this method adds compared to the original...
"""
```

Which should resolve to something like this iirc:

> See [update_or_create](https://docs.djangoproject.com/en/2.0/ref/models/querysets/#update-or-create).
>
> Explain what this functionality this method adds compared to the original...

Reply:
Done